script/configure-integration-tests.ps1

```diff
@@ -1,4 +1,4 @@
-﻿# TODO: this should indicate whether a variable is required or optional
+# TODO: this should indicate whether a variable is required or optional
```

Comment:
This is an old comment from before I actually implemented the optional parameters. Feel free to remove this :fire: